utility-belt/src/main/java/io/confluent/admin/utils/cli/TopicEnsureCommand.java

```diff
@@ -99,17 +103,17 @@ public static void main(String[] args) {
     } catch (ArgumentParserException e) {
       if (args.length == 0) {
         parser.printHelp();
-        success = true;
+        validationSuccess = true;
       } else {
         parser.handleError(e);
-        success = false;
+        validationSuccess = false;
       }
     } catch (Exception e) {
       log.error("Error while running topic-ensure {}.", e);
-      success = false;
+      validationSuccess = false;
     }
 
-    if (success) {
+    if (validationSuccess) {
```

Comment:
Shouldn't this also still check `success`? There's a code path that updates success but doesn't do anything with `validationSuccess`.

And is this condition even correct? The only path that currently sets `validationSuccess` to true is via an exception handler...

Reply:
I just want to keep a `flag` if any of the topics mismatches with the actual topic spec, right now the code only exits with termination state if the last topic mismatches, as success will be set to true if the last topic matches with the actual topic spec and other topics may not match with the actual topic spec.

so I wanted to add a `flag` so if any of the topic spec mismatches then we can exit with termination state. This was the bug stated in the JIRA - https://confluentinc.atlassian.net/browse/LOGGING-308.

Reply:
I modified the code, removed `validationSuccess` variable and added a break statement if validation of topic returns false. @ewencp please review.